app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    move_command, from_file, to_file = command.split()
+    with open(from_file) as fr_f:
+        content = fr_f.read()
+    dirs = to_file.split("/")
+    final_name = dirs.pop(-1)
+    path = ""
+    for directory in dirs:
+        path = os.path.join(path, directory)
+        if not os.path.exists(path):
+            os.mkdir(path)
```

Comment:
U can do this in two line with ```os.path.split``` and ```os.makedirs``` methods

Reply:
os.path.split could split path into "head" and "tail", but I anyway need to move through the path and check if does first directory in the path exist, and then check one-by-one directories. Of course, I still can use os.path.split, but I should split the path in the "head" part and make all the steps to check, which I explain before. But maybe I don't understand your proposal, sry